packages/react/types/css-prop.d.ts

```diff
@@ -0,0 +1,9 @@
+import {} from 'react'
```

Comment:
This file can be included manually by people not wanting to use `@jsx` pragma. Those types here are slightly worse than the ones provided for `@jsx` pragma but AFAIK there is no way to override a type alias already defined in a namespace so I can't redefine `LibraryManagedAttributes` to only allow `css` prop for components already accepting `className` prop.